demo/c/test_demo.py

```diff
@@ -0,0 +1,143 @@
+#
+# Copyright 2021 Picovoice Inc.
+#
+# You may not use this file except in compliance with the license. A copy of the license is located in the "LICENSE"
+# file accompanying this source.
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+# an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+#
+
+import logging
+import os
+import platform
+import subprocess
+import sys
+import unittest
+
+from pvporcupine import util
+
+class PorcupineDemoTestCase(unittest.TestCase):
+    @staticmethod
+    def __has_detected(s):
+        return "detected" in s
```

Comment:
we should probably check if it was detected exactly `n` times (`n` being an input argument). If the engine goes off 3 times on a file with only 1 instance of the keyword, we have a problem.